apps/api/src/app/shared/services/cache/invalidate-cache.service.ts

```diff
@@ -1,43 +1,12 @@
 import { CacheService } from './cache.service';
 import { buildKey, CacheInterceptorTypeEnum } from '../../interceptors';
 import { Injectable, Logger } from '@nestjs/common';
-import { CacheKeyPrefixEnum } from './keys';
+import { CacheKeyPrefixEnum } from './key-builders/shared';
 
 @Injectable()
 export class InvalidateCacheService {
   constructor(private cacheService: CacheService) {}
 
-  public async clearCache({
-    storeKeyPrefix,
-    credentials,
-  }: {
-    storeKeyPrefix: CacheKeyPrefixEnum | CacheKeyPrefixEnum[];
-    credentials: { _id: string; _environmentId: string } | Record<string, unknown>;
-  }) {
-    Logger.log('Clearing the cache of keys with the specified prefixes');
-    Logger.debug('StoreKeyPrefix(s) are: ' + storeKeyPrefix);
-    Logger.debug('Credentials are: ' + (credentials._id as string));
-    if (!this.cacheService?.cacheEnabled()) {
-      Logger.verbose('Cashing service is not enabled to clear Cache.');
-
-      return;
-    }
-
-    if (Array.isArray(storeKeyPrefix)) {
-      Logger.verbose('Mapping all keys to flush');
-      const invalidatePromises = storeKeyPrefix.map((prefix) => {
-        return this.clearByPattern(prefix, credentials);
-      });
-
-      Logger.debug('invalidate promises are: ' + invalidatePromises);
-      Logger.verbose('Removing all keys with prefix');
-      await Promise.all(invalidatePromises);
-    } else {
-      Logger.warn('StoreKeyPrefix is not in array format');
-      await this.clearByPattern(storeKeyPrefix, credentials);
-    }
-  }
-
```

Comment:
same here removed this function so no one will use the evil scan on the cache by mistake.
I still left the 'clearByPattern' function because it is private, I may delete it in near future.